### PR TITLE
Only fade in bookmark on direct page loads

### PR DESCRIPTION
**Pull request by adamschwartz**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -25,13 +27,6 @@ const normalizeCollection = (collection, features, projects) => {
   return normalized
 }
 
-// h/t https://react.30secondsofcode.org/snippet/useSSR
-const isDOMavailable = !!(
```
Comment (signalnerve) [inline src/utils.js]: any reason why we wouldn't just do `export const isDOMavailable`? if you want to move it into a better filename (isDOMAvailable) maybe we can do `src/utils/isDOMavailable.js` (and move other things in a separate PR)

Comment (adamschwartz) [reply src/utils.js]: > any reason why we wouldn't just do `export const isDOMavailable`?

The conventions established by the repo so far are that we keep the export to the last line only, e.g. [`sanity/schemas/project.js`](https://github.com/cloudflare/built-with-workers/blob/b19bca99cec78146bb739fd9f72e7b4c164d07c2/sanity/schemas/project.js#L57). That being said, I’m happy to change this convention if you prefer to write it the other way.

> if you want to move it into a better filename (isDOMAvailable) maybe we can do `src/utils/isDOMavailable.js` (and move other things in a separate PR)

I’d be fine to restructure it into a `utils/` directory. The reason for doing it in this PR is because this PR introduces the second use of the constant, so there wouldn’t have really been a need to break it out prior to this. That being said, I don’t mind breaking it up into two separate PRs for clarity if that’s what you’re suggesting.

Comment (signalnerve) [reply src/utils.js]: cool! makes sense to me. i'm ok w/ doing any moving/reorganizing of code in a separate PR. +1 on this and will merge it so we're not blocked on improving this functionality
